packages/vapor/resources/icons/svg/availability.svg

```diff
@@ -0,0 +1,7 @@
+<svg fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<circle cx="8" cy="19" r="1" stroke="#282829"/>
+<circle cx="16" cy="19" r="1" stroke="#282829"/>
+<path d="M18.5 16.5H7.5C6.39543 16.5 5.5 15.6046 5.5 14.5V8.5C5.5 7.94772 5.05228 7.5 4.5 7.5H3.5" stroke="#282829" stroke-linecap="round"/>
```

Comment:
`stroke="#282829"` must be set only on the `svg` tag, otherwise we won't be able to change its color.

Same comment for the 3 svgs

Reply:
Thanks, will fix it!

Reply:
Does the fill on line 6 of availability.svg need to be altered as well? If so, would you have a hint as to how I would do it? I'm not too familiar with SVGs. Thanks :)